extensions/sql-migration/src/constants/strings.ts

```diff
@@ -264,3 +264,11 @@ export const TARGET_AZURE_SQL_INSTANCE_NAME = localize('sql.migration.target.azu
 export const CUTOVER_TYPE = localize('sql.migration.cutover.type', "Cutover type");
 export const START_TIME = localize('sql.migration.start.time', "Start Time");
 export const FINISH_TIME = localize('sql.migration.finish.time', "Finish Time");
+
+//Source Credentials page.
+export const SOURCE_CONFIGURATION = localize('sql.migration.source.configuration', "Source Configuration");
+export const SOURCE_CREDENTIALS = localize('sql.migration.source.credentials', "Source Credentials");
+export function ENTER_YOUR_SQL_CREDS(sqlServerName: string) {
+	return localize('sql.migration.enter.your.sql.creds', "Enter the credentails for source SQL server instance ‘{0}’", sqlServerName);
+}
+export const USERNAME = localize('sql.migratin.username', "Username");
```

Comment:
migration

I'd suggest installing a spellcheck extension in VS Code if you don't have one already to help avoid these kinds of errors (even if this one doesn't actually affect anything it's still good to keep things clean)